### PR TITLE
Refactor/modify repository defaults

### PR DESCRIPTION
**Pull request by Judge40**

### CODE REVIEW

**Comment by Judge40** [comment]
It seems sensible enough to ignore the quality gate failures given this is essentially the initial commit.

File: .github/workflows/ci-cd-workflow.yml [hunk 1]
```diff
@@ -166,149 +166,150 @@ jobs:
...
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-west-2
+
+      - name: Log in to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Create pre-prod task definition
```
Comment (DorisWongNHS) [inline .github/workflows/ci-cd-workflow.yml:298]: `prod` instead of `pre-prod`?

File: .github/workflows/ci-cd-workflow.yml [hunk 2]
```diff
@@ -166,149 +166,150 @@ jobs:
...
+        run: sed 's/${environment}/preprod/g' .aws/task-definition-template.json > .aws/task-definition-preprod.json
+
+      - name: Fill in the new image ID in the Amazon ECS task definition
+        id: task-def
+        uses: aws-actions/amazon-ecs-render-task-definition@v1
+        with:
+          task-definition: .aws/task-definition.json
```
Comment (DorisWongNHS) [inline .github/workflows/ci-cd-workflow.yml:256]: Wondering does it need to be `.aws/task-definition-preprod.json`?

**Comment by DorisWongNHS** [comment]
Love the generation of task-definitions for both environments from the pipeline 🌮

**Review by DorisWongNHS — APPROVED** [review]

**Pull request merged**